pkg/gadgets/run/tracer/helpers.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tracer
+
+import (
+	"fmt"
+
+	"github.com/cilium/ebpf/btf"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/run/types"
+)
+
+// getAnyMapElem returns the first element of a map. If the map is empty, it returns nil, nil.
+func getAnyMapElem[K comparable, V any](m map[K]V) (*K, *V) {
```

Comment:
I still feel uneasy about this function. AFAIU there is no guarantee that in the following example
```golang
k1, v1 = getAnyMapElem(myMap)
k2, v2 = getAnyMapElem(myMap)
```
`k1 == k2 && v1==v2` is true

I know we are checking somewhere in the code that there is only a single tracer map  - and that is what we currently support. So its okay

Reply:
I agree that's not totally clear, but TBH I don't know how to improve it right now. Hopefully the limitation of a having a single tracer will be removed soon and we don't need to use it anymore.